dotcms-integration/src/test/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjaxTest.java

```diff
@@ -519,10 +520,9 @@ public void test_searchContentletsByUser_shouldReturnContentWithMultiLang() thro
 		Map<String,Object> result = (Map<String,Object>)results.get(0);
 		Assert.assertEquals((Long)result.get("total"), Long.valueOf(2));
 		// Validate the two different languages
-		result = (Map<String,Object>)results.get(3);
-		assertTrue(Long.parseLong(String.valueOf(result.get("languageId")))==language.getId());
-		result = (Map<String,Object>)results.get(4);
-		assertTrue(Long.parseLong(String.valueOf(result.get("languageId")))==defaultLang.getId());
+		final long contentletOne_Language = Long.parseLong(((Map<String,String>)results.get(3)).get("languageId"));
```

Comment:
I think this removes some of the specifics the original was testing.   It the only problem that we should have an entry with language.get() and defaultLang.getId() but they could be reversed.   It is now changed to just see if they are different which should work, but should we be checking they actually match.  If the order is what is breaking it here then something like the following may work to test both cases.
Map<String, Object> result = (Map<String, Object>) results.get(0);
Assert.assertEquals((Long) result.get(“total”), Long.valueOf(2));
// Attempt to identify which result corresponds to each language
Map<String, Object> resultAt3 = (Map<String, Object>) results.get(3);
Map<String, Object> resultAt4 = (Map<String, Object>) results.get(4);
long languageIdAt3 = Long.parseLong(String.valueOf(resultAt3.get(“languageId”)));
long languageIdAt4 = Long.parseLong(String.valueOf(resultAt4.get(“languageId”)));
// Assuming language and defaultLang are predefined variables with their respective IDs
if (languageIdAt3 == language.getId()) {
    assertTrue(languageIdAt4 == defaultLang.getId());
} else if (languageIdAt3 == defaultLang.getId()) {
    assertTrue(languageIdAt4 == language.getId());
} else {
    throw new AssertionError(“Neither result matches the expected language IDs.“);
}

Reply:
The goal of this test is to check that it pulls content from different languages, it doesn't matter the order of them, so that's why I changed just to check that.